lockfile.json

```diff
@@ -354,7 +354,7 @@
     "revision": "962568c9efa71d25720ab42c5d36e222626ef3a6"
   },
   "inko": {
-    "revision": "aecabede39b0db05678e2d4686258d4f71b00a51"
+    "revision": "a6fb0b0bfee5dede688968fd91282da799e5aa8e"
```

Comment:
If preferred, I can just wait until the bot updates the lockfile and then rebase this PR.

Reply:
This is fine, thank you (keeps changes atomic). I'm trying to move away from the constant update churn towards more meaningful updates.